tests/PHPUnit/Unit/DeprecatedMethodsTest.php

```diff
@@ -82,8 +82,6 @@ public function test_deprecations()
         $this->assertDeprecatedMethodIsRemovedInPiwik4('Piwik\Updates', 'getMigrationQueries');
         $this->assertDeprecatedMethodIsRemovedInPiwik4('Piwik\Updater', 'executeMigrationQueries');
 
-        // THIS IS A REMINDER FOR PIWIK 4: We need to rename getColumnType() to getDbColumnType() and $columnType to $dbColumnType
-        $this->assertDeprecatedMethodIsRemovedInPiwik4('Piwik\Columns\Dimension', 'getType');
```

Comment:
Looked into this for a while. I can kind of see why I had this thought initially, it doesn't really help much though and it's not needed to remove it or change anything.